### DISCUSSION TITLE
trying out this feature to see if  can leave slack completely

### DISCUSSION THREAD

**Discussion by 08wparker** [Common-Longitudinal-ICU-data-Format/CLIF#27] [General]
I want us to be more efficient about "locking in" ETL decisions we make in Thursday PM call. Right now, Slack is only a place for informal discussion. If we could replicate that here, then we don't need the Slack. The idea is people would be more likely to use "issues".

However, I do worry that not everyone has a GitHub account and dropping Slack completely may be too exclusionary

**Comment by 08wparker** [comment]
@sajor2000 @cloverbunny @ingra107

**Reply by ingra107** [reply]
As long as ppl have mentions set up to their email this could work... but agree... not sure how savvy everyone is with github.  I am still learning to be honest!
